pkg/antctl/raw/featuregates/command.go

```diff
@@ -87,19 +87,25 @@ func featureGateRequest(cmd *cobra.Command, mode string) error {
 		return err
 	}
 	var agentGates []featuregates.Response
+	var agentWindowsGates []featuregates.Response
 	var controllerGates []featuregates.Response
 	for _, v := range resp {
-		if v.Component == "agent" {
+		switch v.Component {
```

Comment:
I don't think we should conflate "component" for which we are receiving feature gates values and antctl runtime modes, even if the string values happen to overlap / be the same.

Maybe you could export the following constants in `pkg/apiserver/handlers/featuregates/handler.go`, instead of keeping them private:
```go
	AgentMode            = "agent"
	AgentWindowsMode     = "agent-windows"
	ControllerMode       = "controller"
```

After all, this file already imports `antrea.io/antrea/pkg/apiserver/handlers/featuregates`.